src/agent/coverage/src/lib.rs

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-
-#![cfg(windows)]
 #![allow(clippy::as_conversions)]
 #![allow(clippy::new_without_default)]
 
+#[macro_use]
+extern crate serde;
```

Comment:
I don't think this is still necessary.

Reply:
I was using it to bring the `derive` proc macros in scope throughout. That said, I just switched to module-local imports instead.